man/import.q.concourse.Rd

```diff
@@ -28,13 +31,18 @@
 	This function simply imports all full item wordings and assigns a \emph{meaningful item short handle} based on the file name.
```

Comment:
I've now (see above) turned (almost all, except in directly following sentences) mentions of `handles` into (short, meaningful) `handles`.
That *is* redundant, but I hope to make sure that people don't confuse handles with IDs that way.